tldw_Server_API/app/core/Setup/install_manager.py

```
@@ -1069,6 +1070,33 @@ def _install_kokoro(variants: list[str]) -> None:
         _download_hf_dir(onnx_repo, 'voices', voices_dir)
 
 
+def _install_kitten_tts(variants: list[str]) -> None:
+    _ensure_downloads_allowed('KittenTTS model assets')
+    try:
+        from tldw_Server_API.app.core.TTS.vendors.kittentts_compat import download_model_assets
+    except Exception as exc:  # noqa: BLE001
+        raise RuntimeError('KittenTTS compatibility runtime is required for asset downloads.') from exc
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)